src/pages/tokenDrop/Congratulations.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Stack, useMediaQuery, useTheme } from "@mui/material";
+import { useWindowDimensions } from "common";
+import {
+  GradientTypography,
+  OutlinedButtonNoGradient,
+  Typography,
+} from "elements";
+import { FunctionComponent } from "react";
+import { useNavigate } from "react-router-dom";
+import Confetti from "react-confetti";
+
+const Congratulations: FunctionComponent = () => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  const window = useWindowDimensions();
+  const isMdScreen = useMediaQuery(
+    `(min-width:${theme.breakpoints.values.md}px)`
```

Comment:
what does the "min-width:" do in this case? I have used the following method to try to figure out the breakpoint and trying to figure out if this is just another way to do the same thing:

`const isBelowMediumBreakpoint = useMediaQuery(theme.breakpoints.down("md"));`